src/router/index.js

```diff
@@ -77,6 +77,15 @@ export default new Router({
 				showShared: true,
 			}),
 		},
+		{
+			path: '/videos',
+			component: Timeline,
+			name: 'videos',
+			props: route => ({
+				rootTitle: t('photos', 'Your videos'),
+				onlyVideos: true,
```

Comment:
we could also directly pass the mimes as props from here?

Reply:
Something like this ?
```suggestion
				mimesType: videoMimes,
```

Reply:
good idea

Reply:
That way you can even do the spread from here:
```suggestion
				mimesType: [...videoMimes, ...imageMimes],
```

Reply:
gotcha